setup.py

```diff
@@ -15,6 +15,7 @@
 from setuptools import setup
 
 install_reqs = [
+    'mock>=2.0.0',
```

Comment:
While I've historically put `mock` under `install_reqs`, these days we can just make it a test dependency right? @posita has done a good job of making that separation clear.